Plugin/AddOptionCode.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace SnowIO\AttributeOptionCode\Plugin;
+
+use Magento\Catalog\Setup\CategorySetup;
+use Magento\Framework\Registry;
+use SnowIO\AttributeOptionCode\Api\AttributeOptionCodeRepositoryInterface;
+use Magento\Eav\Model\Entity\Attribute\AbstractAttribute;
+use SnowIO\AttributeOptionCode\Block\Adminhtml\Attribute\Edit\Options\Visual;
+use SnowIO\AttributeOptionCode\Block\Adminhtml\Attribute\Edit\Options\Text;
+use SnowIO\AttributeOptionCode\Block\Adminhtml\Attribute\Edit\Options\Options;
+
+class AddOptionCode
+{
+    /** @var Registry */
+    protected $registry;
+
+    /** @var AttributeOptionCodeRepositoryInterface */
+    private $attributeOptionCodeRepository;
+
+    /**
+     * AddOptionCode constructor.
+     * @param AttributeOptionCodeRepositoryInterface $attributeOptionCodeRepository
+     * @param Registry $registry
+     */
+    public function __construct(
+        AttributeOptionCodeRepositoryInterface $attributeOptionCodeRepository,
+        Registry $registry
+    ) {
+        $this->attributeOptionCodeRepository = $attributeOptionCodeRepository;
+        $this->registry = $registry;
+    }
+
+    /**
+     * @param Visual|Text|Options $subject
+     * @param array $result
+     * @return array
+     */
+    public function afterGetOptionValues($subject, array $result)
+    {
+        $this->addOptionCodeToOptionValuesData($result);
+        return $result;
+    }
+
+    /**
+     * @param $values
+     * @return array
+     */
+    private function addOptionCodeToOptionValuesData(array $values): array
+    {
+        $ids = $this->getIdsFromValuesData($values);
+        $attributeCode = $this->getAttributeObject() ? $this->getAttributeObject()->getAttributeCode() : null;
```

Comment:
So this could be simplified. If `getAttributeObject` could return `null` or the object we could do:
```php
$attributeObject = $this->getAttributeObject();
if (!$attributeObject) {
    return $values;
}
...
$optionCodes = $this->getOptionCodes($attributeObject->getAttributeCode(), $ids);